docs/guides/functions/connecting-a-rest-api.md

```diff
@@ -3,6 +3,8 @@ title: Connecting a REST API to a Lambda function
 description: How to connect a REST API to a Lambda function
 ---
 
+<inline-fragment platform="flutter" src="~/guides/fragments/flutter/flutterWarning.md"></inline-fragment>
```

Comment:
I believe that the content under guides/functions is NOT API category, but the functions category.